test/verify/check-machines

```diff
@@ -212,6 +212,8 @@ class TestMachines(MachineCase):
         b.wait_present("#test-vm-action-{0}".format(self.providerVm1_name)) # is provider-specific action button rendered?
         b.wait_in_text("#test-vm-props-{0}".format(self.providerVm1_name), "Test Provider Property") # are provider-specific VM props rendered?
 
+        b.click("{0}-sendNMI".format(self.providerVm1_id)) # click the Send Non-Maskable Interrupt button
```

Comment:
In addition to @petervo 's comment , the VM configuration within these check-machines tests is given. 
Does the NMI affect the test machine in its particular configuration somehow? We can verify the outcome or some (deterministic) side-effect.
If the recently used test VM is "immune" to the NMI, can we change the VM configuration somehow so the NMI takes effect? 
Let's keep in mind, we are not testing Libvirt but just the start-to-end flow within cockpit-machines.